src/components/Users/InvitesTable.js

```diff
@@ -0,0 +1,90 @@
+import styled from '@emotion/styled';
+import { relativeDate } from 'lib/helpers';
+import PropTypes from 'prop-types';
+import React from 'react';
+import BootstrapTable from 'react-bootstrap-table-next';
+
+const TableWrapper = styled.div`
+  .react-bootstrap-table table {
+    table-layout: auto;
+  }
+
+  .status {
+    width: 90px;
+  }
+
+  .actions {
+    width: 100px;
+  }
+`;
+
+const tableDefaultSorting = [
+  {
+    dataField: 'email',
+    order: 'asc',
+  },
+];
+
+// Provides the configuraiton for the clusters table
+const getTableColumnsConfig = () => {
+  return [
+    {
+      classes: 'email',
+      dataField: 'email',
+      text: 'Email',
+      sort: true,
+    },
+    {
+      classes: 'created',
+      dataField: 'created',
+      text: 'INVITED',
+      sort: true,
+      formatter: relativeDate,
+    },
+    {
+      classes: 'invited_by',
+      dataField: 'invited_by',
+      text: 'INVITED BY',
+      sort: true,
+    },
+    {
+      classes: 'expiry',
+      dataField: 'expiry',
+      text: 'EXPIRES',
+      sort: true,
+      formatter: relativeDate,
+    },
+  ];
+};
+
+// eslint-disable-next-line react/no-multi-comp
+const InvitesTable = ({ invitations }) => {
```

Comment:
Do you mind bringing a bit of TypeScript into this file? 😬

Reply:
In fact I do mind... but I guess I'll have to.

Reply:
I tried, but failed with this problem:

![image](https://user-images.githubusercontent.com/273727/97588005-54cb6180-19fc-11eb-9338-96494aa64ab4.png)